website/templates/emails/contributor_added.txt.mako

```diff
@@ -0,0 +1,15 @@
+Hello ${user.fullname},
+
+You have been added as a contributor to the project "${node.title}" on the Open Science Framework: ${node.absolute_url}
+
+If you are erroneously being associated with "${node.title}" then you may visit the project sharing page and remove yourself as a contributor.
```

Comment:
That's what I had before, but when I moved the `contrib_added` signal to `Node.add_contributor`, it caused so many tests to fail. In these tests, `Node.add_contributor` gets called before the node is saved for the first time. This causes an error when building the sharing page url for the email because the node._id is None. @lyndsysimon suggested removing it as the simple solution.